pkg/ccl/backupccl/datadriven_test.go

```diff
@@ -101,7 +101,7 @@ func newDatadrivenTestState() datadrivenTestState {
 	}
 }
 
-func (d *datadrivenTestState) cleanup(ctx context.Context) {
+func (d *datadrivenTestState) cleanup(ctx context.Context, t *testing.T) {
```

Comment:
why do we need t as a param? I don't think we're using it.

Reply:
oops. it's necessary. check out the latest push.